src/init.cpp

```diff
@@ -634,6 +634,8 @@ void SetupServerArgs(ArgsManager& argsman)
                              "is of this size or less (default: %u)",
                              MAX_OP_RETURN_RELAY),
                    ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
+    argsman.AddArg("-relayinscription", strprintf("Relay and mine inscription transactions and remove the fee reduction for them (default: %u)", DEFAULT_RELAY_INSCRIPTION), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
+    argsman.AddArg("-relayinscriptionsize", strprintf("Maximum size of inscription transactions we relay and mine (default: %u)", MAX_INSCRIPTION_RELAY), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
```

Comment:
This isn't just about relaying, it's about accepting into your mempool at all -- if someone sends you the result of an inscribed tx you won't be able to spend those funds until they confirm, and (obviously) you won't include them in blocks.

I'd suggest something more like `-limitdummyscriptdatasize`, which also avoids tying it to one particular protocol; given inscriptions could change their spec to use other formats, and other protocols could adopt a similar format, a more generic name seems better.

Reply:
good idea, I've change the code to have the `-limitdummyscriptdatasize` option.